pkg/host/host_linux.go

```diff
@@ -52,23 +55,35 @@ func isSupported(c *prog.Syscall, sandbox string) (bool, string) {
 	// Kallsyms seems to be the most reliable and fast. That's what we use first.
 	// If kallsyms is not present, we fallback to execution of syscalls.
 	kallsymsOnce.Do(func() {
-		kallsyms, _ = ioutil.ReadFile("/proc/kallsyms")
+		kallsyms, _ := ioutil.ReadFile("/proc/kallsyms")
+		if len(kallsyms) == 0 {
+			return
+		}
+		res := []string{
+			` T sys_([^\n]+)\n`, ` T ksys_([^\n]+)\n`,
```

Comment:
I wonder if it will be faster to combine this all into a single regexp and do just 1 pass? It should be.